QtCollider/widgets/QcWebView.h

```diff
@@ -58,13 +58,12 @@ class WebView : public QWebEngineView {
     Q_INVOKABLE void navigate(const QString& url);
 
 public Q_SLOTS:
-    void findText(const QString& searchText, bool reversed, QtCollider::QcCallback* cb);
```

Comment:
IIRC the namespace here is intentional -- see line 49 above -- why did you remove it, and are you sure this works without it?

Reply:
i don't think you need the default argument either